.github/workflows/tarpaulin.yml

```diff
@@ -0,0 +1,66 @@
+name: tarpaulin
+
+on:
+  pull_request:
+    paths:
+      - Cargo.lock
+      - Cargo.toml
+      - src/*.rs
+      - src/**/*.rs
+      - src/**/**/*.rs
+
+permissions:
```

Comment:
Tarpaulin needs to be able to access the repository's contents, so we need reading permissions for the source files.  In order to post a comment, we need writing permissions for PRs.